R/catboost.R

```diff
@@ -194,6 +194,27 @@ add_boost_tree_catboost <- function() {
   )
 }
 
+prepare_df_catboost <- function(x, y = NULL, categorical_cols= NULL) {
```

Comment:
Shoudn't we call prepare_df_catboost in `predict.catboost.Model` too?

Reply:
I am sooo glad there are multiple eyes on this!

Reply:
This calls for more tests 👀

Reply:
I wish the resulting data structure in lightgbm and catboost was more inspectable from R

Reply:
yeah, it would much easier! It seems to be just a pointer to some C data structure...